Hmm... something is wrong with the build of rdm.cpp. I am getting a strange compile error on OSX:
```
[ 86%] Building CXX object src/CMakeFiles/rdm.dir/rdm.cpp.o
In file included from /private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/rdm.cpp:32:
/private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/RTags.h:978:5: warning: default label in switch which covers all enumeration values [-Wc
overed-switch-default]                                                                                                                                                              
    default:
    ^
/private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/rdm.cpp:318:11: error: use of undeclared identifier 'FileManagerWatch'; did you mean 'No
FileManagerWatch'?                                                                                                                                                                  
        { FileManagerWatch, "filemanager-watch", 'M', CommandLineParser::NoValue, "Use a file system watcher for filemanager." },
          ^~~~~~~~~~~~~~~~
          NoFileManagerWatch
/private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/rdm.cpp:184:5: note: 'NoFileManagerWatch' declared here
    NoFileManagerWatch,
    ^
/private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/rdm.cpp:585:14: error: use of undeclared identifier 'FileManagerWatch'; did you mean 'No
FileManagerWatch'?                                                                                                                                                                  
        case FileManagerWatch: {
             ^~~~~~~~~~~~~~~~
             NoFileManagerWatch
/private/var/folders/tv/h06nsg4s31b9lcd0_wl1rcjh0000gn/T/nix-build-rtags-2.8.drv-0/rtags/src/rdm.cpp:184:5: note: 'NoFileManagerWatch' declared here
    NoFileManagerWatch,
    ^
1 warning and 2 errors generated.
```
Afaik the preprocessor conditionals on https://github.com/Andersbakken/rtags/blob/master/src/rdm.cpp#L39 and https://github.com/Andersbakken/rtags/blob/master/src/rdm.cpp#L181 are quite strange combination to me. Any idea?

--- a/pkgs/development/tools/rtags/default.nix
+++ b/pkgs/development/tools/rtags/default.nix
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   name = "rtags-${version}";
-  version = "2.3";
+  version = "2.8-p1";
 
   buildInputs = [ cmake llvmPackages.llvm openssl llvmPackages.clang emacs ]
-    ++ lib.optional stdenv.isDarwin apple_sdk.sdk;
+    ++ lib.optionals stdenv.isDarwin [ apple_sdk.sdk apple_sdk.frameworks.CoreServices ];
 
   preConfigure = ''
-    export LIBCLANG_CXXFLAGS="-isystem ${llvmPackages.clang.cc}/include $(llvm-config --cxxflags) " \
+    export LIBCLANG_CXXFLAGS="-isystem ${llvmPackages.clang.cc}/include $(llvm-config --cxxflags) -fexceptions" \
            LIBCLANG_LIBDIR="${llvmPackages.clang.cc}/lib" \
 
   '' + lib.optionalString stdenv.isDarwin ''
@@ -17,10 +17,11 @@ stdenv.mkDerivation rec {
   '';
 
   src = fetchgit {
-    rev = "refs/tags/v${version}";
+    # rev = "refs/tags/v${version}"; # TODO Renable if sha1 below is tagged as release
+    rev = "f85bd60f00d51748ea159b00fda7b5bfa78ef571";
     fetchSubmodules = true;
     url = "https://github.com/andersbakken/rtags.git";
-    sha256 = "05kzch88x2wiimygfli6vsr9i5hzgkybsya8qx4zvb6daip4b7yf";
+    sha256 = "0g9sgc763c5d695hjffhis19sbaqk8z4884szljf7kbrjxl17y78";
   };
 
   meta = {

